plugins/rapid7_insightidr/help.md

```diff
@@ -40,6 +47,411 @@ Example input:
 
 ### Actions
 
+#### List Alerts for Investigation
+
+This action is used to retrieve a page of alerts associated with the specified investigation.
+
+##### Input
+
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|id|string|None|True|The ID or RRN of the investigation|None|rrn:investigation:example:11111111-1111-1111-1111-111111111111:investigation:11111111|
+|index|integer|0|True|The optional zero-based index of the page to retrieve. Must be an integer greater than or equal to 0|None|1|
```

Comment:
The description says optional, but the field is marked as required. Can you please update the description accordingly